### PR TITLE
fix(deps): Bump gravitee-reporter-api to version 1.31.2

### PR DESCRIPTION
**Pull request by dev-marek**
**Description**

Update gravitee-reporter-api

**Additional context**

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `3.5.2-fix-deps-bump-gravitee-reporter-api-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/gateway/gravitee-gateway-api/3.5.2-fix-deps-bump-gravitee-reporter-api-SNAPSHOT/gravitee-gateway-api-3.5.2-fix-deps-bump-gravitee-reporter-api-SNAPSHOT.zip)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -120,6 +120,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
+            <version>1.18.32</version>
```
Comment (ThibaudAV) [inline pom.xml:123]: why you add this version ?

Comment (ThibaudAV) [reply pom.xml:123]: it is "carried" by the parent so you should be able to delete it here 🤔

Comment (dev-marek) [reply pom.xml:123]: :O My mistake

**Comment by dev-marek** [comment]
Related to: https://github.com/gravitee-io/gravitee-reporter-api/pull/87 as per conversation with @ThibaudAV

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**